pkg/utils/common/controller.go

```diff
@@ -634,3 +635,34 @@ func (s *SuiteController) CreateRoleBinding(roleBindingName, namespace, subjectK
 	}
 	return createdRoleBinding, nil
 }
+
+// CreateRegistryAuthSecretKeyVlaue creating key/value kubernetes secret.
```

Comment:
I think the values needing to be decoded may be a unique situation. Maybe reflect that in the function name somehow?

Reply:
added Decoded to end of name